web/third/bootstrap-typeahead/typeahead.js

```diff
@@ -199,36 +199,38 @@ import {get_string_diff} from "../../src/util";
   }
 
   , show: function () {
+    if (!this.options.doNotMove) {
```

Comment:
Having a boolean optional parameter, true by default, named `manuallyPosition` instead, might be clearer.

Reply:
I actually ended up pairing this with `parentElement` since I think the two will be always paired together. Does that seem reasonable to you?

Reply:
Makes sense to me!

Even if we have a usecase in the future where we manually position inside a parent element, we can cross that bridge when we come to it.